tests/e2e/functional_test.go

```diff
@@ -251,13 +291,13 @@ var _ = Describe("Functional e2e", Serial, func() {
 			return rollout, nil
 		})
 
-		// TODO: check that when pipeline is being updated, it is being paused
-		//       could set envVar to signify dataLossPrevention is on
-		// Eventually(func() metav1.ConditionStatus {
-		// 	rollout, _ := pipelineRolloutClient.Get(ctx, pipelineRollouName, metav1.GetOptions{})
-		//
-		// return getRolloutCondition(rollout.Status.Conditions, apiv1.ConditionPipelinePausingOrPaused)
-		// }, testTimeout, testPollingInterval).Should(Equal(metav1.ConditionTrue))
+		if dataLossPrevention == "true" {
+			document("Verify that child Pipeline is paused by checking rollout condition")
+			Eventually(func() metav1.ConditionStatus {
+				rollout, _ := pipelineRolloutClient.Get(ctx, pipelineRolloutName, metav1.GetOptions{})
+				return getRolloutCondition(rollout.Status.Conditions, apiv1.ConditionPipelinePausingOrPaused)
+			}, testTimeout, testPollingInterval).Should(Equal(metav1.ConditionTrue))
```

Comment:
maybe we should actually reduce the interval for this one, since the value will go from True to False, we need to make sure we catch it.

Reply:
Could we use the default 10ms for just these cases then?

Reply:
let's do that for any test that's checking that pausing=true